src/predefinedSimulations.jl

```diff
@@ -11,15 +11,16 @@ predef_eeg(;kwargs...)
 predef_eeg(rng;kwargs...)
 predef_eeg(rng,n_subjects;kwargs...)
 
-Gene
-rates a P1/N1/P3 complex.
-In case `n_subjects` is defined - `MixedModelComponents`` are generated, else `LinearModelComponents`
+Generates a P1/N1/P3 complex.
+In case `n_subjects` is defined - `MixedModelComponents` are generated, else `LinearModelComponents`
+
+Most used `kwargs` is: `return_epoched=true` to ignore the overlap/onset bits and return already epoched data
```

Comment:
```suggestion
Generate a P1/N1/P3 complex.
In case `n_subjects` is defined - `MixedModelComponents` are generated, else `LinearModelComponents`.

The most used `kwargs` is: `return_epoched=true` to ignore the overlap/onset bits and return already epoched data.
```

Reply:
But `return_epoched = true` does not necessarily mean that the `onset` is ignored. It would also be possible to have epoched data with overlapping signals.